pythran/tests/cases/multitype.py

```diff
@@ -0,0 +1,6 @@
+#pythran export square(int or float)
+#runas square(200)
+#runas square(200.200)
+
+def square(n):
+    return n*n
```

Comment:
maybe you could take an existing test case that could be rewritten using your notation?